open-machine-learning-jupyter-book/data-science/working-with-data/Indexing_on_ndarrays.ipynb

```diff
@@ -0,0 +1,2166 @@
+{
```

Comment:
<p>This code block is not working, try to fix it</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocademy-ai/machine-learning/pull/608/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>

Reply:
<p>I figured out what was wrong with this block: the original md file wanted to give an example of code that reported an error out of bounds, so the teacher set it as code and actively asked it to report an error, and recorded the error content. I set it and its error reporting to markdown format, and no more errors were reported.</p><p>And I delete the redundant files</p>